src/main/resources/application.properties

```
@@ -4,8 +4,9 @@ spring.datasource.password=****
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.open-in-view=false
 spring.jackson.deserialization.fail-on-unknown-properties=true
-jwt.expiration=300000
+jwt.expiration=30000000000000
 jwt.secret=helloMates1323424dhfkdjhgwjhrgtsidjkbdjk
```

```suggestion
jwt.secret=*******
```
don't push secret word to GH